plugins/xpay/xpay.c

```diff
@@ -1867,7 +1872,8 @@ static struct command_result *json_xpay_params(struct command *cmd,
 	const char *invstring;
 	const char **layers;
 	u32 *maxdelay;
-	unsigned int *retryfor;
+    const char *payer_note;
```

Comment:
nit: To keep the formatting consistent and avoid tabs/spaces alignment issues (especially between editors and the GitHub UI), could we format this to match the existing tab-based style?